ostelco-ios-client/Coordinators/OnboardingCoordinator.swift

```diff
@@ -512,22 +503,34 @@ class RegionOnboardingCoordinator {
             
             UserManager.shared.customer = context.customer
             
-            let region = context.regions
-                .map {
-                    $0.fragments.regionDetailsFragment
-                }
-            .first(where: {
+            let region = context.regions.map {
+                $0.fragments.regionDetailsFragment
+            }.first(where: {
                 $0.region.id == self.region.region.id
             })!
             
-            let stage = self.stageDecider.stageForRegion(region: RegionResponse(gqlData: region), localContext: self.localContext)
+            let stage: StageDecider.RegionStage
+            if let problem = self.checkLocation(country: Country(region.region.id)) {
```

Comment:
@sgoodwin regionID is not a country code, we can't use region id to create countries, this will fail in the future